### ISSUE TITLE
Cannot print report Competitors with rented cards

### ISSUE THREAD

**Issue by paukert** [Quick-Event/quickbox#944]
Runs → Print → Competitors with rented cards

```
2023-08-25T16:24:57[reportviewwidget.cpp:82]|I| Show report: /tmp/.mount_QuickEThYKHm/usr/bin/reports/Runs/qml/reports/competitorsWithCardRent.qml
2023-08-25T16:24:57[reportprocessor.cpp:66]|E| Erorr loading report component: /tmp/.mount_QuickEThYKHm/usr/bin/reports/Runs/qml/reports/competitorsWithCardRent.qml file:///tmp/.mount_QuickEThYKHm/usr/bin/reports/Runs/qml/reports/competitorsWithCardRent.qml:15 SqlTableModel is not a type

QQmlComponent: Component is not ready
2023-08-25T16:24:57[reportprocessor.cpp:115]|E| Error creating root object from component: 0x557cb16e4850 file:///tmp/.mount_QuickEThYKHm/usr/bin/reports/Runs/qml/reports/competitorsWithCardRent.qml
2023-08-25T16:24:57[reportprocessor.cpp:116]|E| Created object: 0
2023-08-25T16:24:57[reportprocessor.cpp:118]|E| file:///tmp/.mount_QuickEThYKHm/usr/bin/reports/Runs/qml/reports/competitorsWithCardRent.qml:15:2: SqlTableModel is not a type
QQmlComponent: Component is not ready
2023-08-25T16:24:57[reportprocessor.cpp:115]|E| Error creating root object from component: 0x557cb16e4850 file:///tmp/.mount_QuickEThYKHm/usr/bin/reports/Runs/qml/reports/competitorsWithCardRent.qml
2023-08-25T16:24:57[reportprocessor.cpp:116]|E| Created object: 0
2023-08-25T16:24:57[reportprocessor.cpp:118]|E| file:///tmp/.mount_QuickEThYKHm/usr/bin/reports/Runs/qml/reports/competitorsWithCardRent.qml:15:2: SqlTableModel is not a type
QQmlComponent: Component is not ready
2023-08-25T16:24:57[reportprocessor.cpp:115]|E| Error creating root object from component: 0x557cb16e4850 file:///tmp/.mount_QuickEThYKHm/usr/bin/reports/Runs/qml/reports/competitorsWithCardRent.qml
2023-08-25T16:24:57[reportprocessor.cpp:116]|E| Created object: 0
2023-08-25T16:24:57[reportprocessor.cpp:118]|E| file:///tmp/.mount_QuickEThYKHm/usr/bin/reports/Runs/qml/reports/competitorsWithCardRent.qml:15:2: SqlTableModel is not a type
```

**Comment by paukert** [comment]
Výše uvedené chyby to už nehází, ale je to zaseklé v nějaké smyčce. V logu se objevily tyhle věci, možná to bude příčina problému.
```
2023-08-27T21:08:53[reportviewwidget.cpp:82]|I| Show report: C:/QuickEvent/reports/Runs/qml/reports/competitorsWithCardRent.qml
file:///C:/QuickEvent/reports/Runs/qml/reports/competitorsWithCardRent.qml:26: TypeError: Cannot read property 'verticalMargin' of undefined
file:///C:/QuickEvent/reports/Runs/qml/reports/competitorsWithCardRent.qml:25: TypeError: Cannot read property 'horizontalMargin' of undefined
file:///C:/QuickEvent/reports/Runs/qml/reports/competitorsWithCardRent.qml:24: TypeError: Cannot read property 'pageHeight' of undefined
file:///C:/QuickEvent/reports/Runs/qml/reports/competitorsWithCardRent.qml:23: TypeError: Cannot read property 'pageWidth' of undefined
```
![obrazek](https://github.com/Quick-Event/quickbox/assets/15065907/13783a8a-464e-4d16-a5d7-e1da3593c534)

**Comment by paukert** [comment]
QBE file was attached to #945.

**Comment by fvacek** [comment]
> Výše uvedené chyby to už nehází, ale je to zaseklé v nějaké smyčce. V logu se objevily tyhle věci, možná to bude příčina problému.
> 
> ```
> 2023-08-27T21:08:53[reportviewwidget.cpp:82]|I| Show report: C:/QuickEvent/reports/Runs/qml/reports/competitorsWithCardRent.qml
> file:///C:/QuickEvent/reports/Runs/qml/reports/competitorsWithCardRent.qml:26: TypeError: Cannot read property 'verticalMargin' of undefined
> file:///C:/QuickEvent/reports/Runs/qml/reports/competitorsWithCardRent.qml:25: TypeError: Cannot read property 'horizontalMargin' of undefined
> file:///C:/QuickEvent/reports/Runs/qml/reports/competitorsWithCardRent.qml:24: TypeError: Cannot read property 'pageHeight' of undefined
> file:///C:/QuickEvent/reports/Runs/qml/reports/competitorsWithCardRent.qml:23: TypeError: Cannot read property 'pageWidth' of undefined
> ```
> 
> ![obrazek](https://user-images.githubusercontent.com/15065907/263554054-13783a8a-464e-4d16-a5d7-e1da3593c534.png)

nemas reports nastaveny na nejakou starsi verzi, mne se to nedari zreprodukovat?

**Comment by paukert** [comment]
Fixed in 3.0.6.
